src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PlacementRequestService.kt

```diff
@@ -296,6 +308,25 @@ class PlacementRequestService(
 
     placementRequestRepository.save(placementRequest)
 
+    placementRequest.booking?.let { booking ->
+      val bookingCancellationResult = bookingService.createCancellation(
+        user,
+        booking,
+        LocalDate.now(),
+        CAS1_WITHDRAWN_BY_PP_ID,
+        "Automatically withdrawn as placement request was withdrawn",
```

Comment:
do you think it would be useful from an auditing and debugging perspective to include the ID of the "initiating" entity here eg.

> Automatically withdrawn as placement request abc123 was withdrawn

Reply:
I agree it would be good to track but capturing here wouldn't fully solve the issue because we have no free text fields on Placement Application and Placement Requests where it would also be good to capture similar information. I think that maybe is a nice to have feature (along with withdrawal date as you suggested on the other PR, which could be used to 'group' withdrawals i guess if we used the exact same snapshot)